udata_front/theme/gouvfr/templates/dataset/display.html

```diff
@@ -397,10 +397,9 @@ <h2 id="community-reuses" class="fr-h2 fr-col">{{ _('Reuses') }} <sup>{{ total_r
         </div>
         <p class="fr-my-0">{{ _('Explore the reuses of this dataset.') }}</p>
         <p class="fr-mt-0 fr-mb-3w">{{ _('Did you use this data ? Reference your work and increase your visibility.') }}</p>
-        <div class="fr-pt-2w fr-pt-md-0 fr-grid-row no-wrap fr-mb-2w overflow-x-scroll">
```

Comment:
Do we want to say bye bye `overflow-x-scroll` or could it get useful later?

Reply:
I removed it. We can add it back later if needed.